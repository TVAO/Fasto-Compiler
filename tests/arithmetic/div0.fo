fun int main() = 
    let x = 10 in 
    let y = 0 in 
    write(x / y)