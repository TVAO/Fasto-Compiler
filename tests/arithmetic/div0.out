Runtime error at line 4: division by zero
